packages/react/src/components/Tag/DismissibleTag.tsx

```diff
@@ -161,6 +167,10 @@ const DismissibleTag = forwardRef(
 
     const dismissLabel = `Dismiss "${text}"`;
 
+    const dismissActionLabel = isEllipsisApplied
+      ? dismissTooltipLabel || dismissLabel
```

Comment:
Should a nullish coalescing operator be used to ensure that falsy values outside of `null` and `undefined` are respected?

```suggestion
      ? dismissTooltipLabel ?? dismissLabel
```

Reply:
Thanks for the suggestion, I intentionally used || instead of ?? to fallback on any falsy value like empty strings, ensuring we avoid blank tooltips in the UI.